src/SquareService.php

```diff
@@ -236,6 +236,7 @@ public function charge(array $data)
                 'amount'   => $data['amount'],
                 'currency' => $currency,
             ],
+            'verification_token' => array_key_exists('verification_token', $data) ? $data['verification_token'] : null,
```

Comment:
To cover the cases where there is no verification token needed, can we instead of setting it's value as null set the attribute itself only if the value for it exists.

Something along these lines:

```php
if (array_key_exists('verification_token', $data)) {
    $prepData['verification_token'] = $data['verification_token'];
}
```

Reply:
Reason for this is that it's bad practice to keep sending null values for attributes used in securing someones identity, especially in financial apps.

Reply:
Yeah, no worries. It's null anyway in the Square `CreatePaymentRequest` so you're still kind of sending null values, unless they strip it themselves, but it's not a problem.